docs/api-reference/next/future/image.md

```diff
@@ -101,6 +101,18 @@ const MyImage = (props) => {
 }
 ```
 
+### fill
+
+A boolean attribute. If present, sets the image to fill its containing element rather than defining its own size. When using `fill` mode, do not provide `height` and `width`.
+
+When using the `fill` attribute, the parent element _must_ have the CSS `position: "relative"` or the image will not render correctly. Also note that the image itself is automatically given `position: "absolute"` and this value should not be changed.
```

Comment:
Technically more correct is that the parent must have a position value that isn't static, right? E.g. `absolute` or `fixed` also work.

Is it good enough to enforce this for the immediate parent? I guess it is less of a foot-gun that way.

Reply:
Fair point. I changed this to:
```When using the `fill` attribute, the parent element _must_ have a non-static CSS [`position`](https://developer.mozilla.org/en-US/docs/Web/CSS/position) attribute, such as `"relative"`, or the image will not render correctly.```